tests/test_loader.py

```diff
@@ -198,6 +199,7 @@ def test_integration_properties(hass):
         },
         {"manufacturer": "Signify", "modelName": "Philips hue bridge 2015"},
     ]
+    assert integration.mqtt == ["hue/discovery"]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6299130)